README.md

```diff
@@ -9,16 +9,38 @@
 
 ## Overview
 
-HealthyHome is an open-source smart home device which monitors the indoors environment conditions, based on a Raspberry Pi.
+HealthyHome is an open-source project centred around a smart home device which monitors the indoor environment conditions of a home.
+
+### Objectives
+
+-   Allow a user to characterise important parameters of their indoor home environment
+-   Provide easy access to both real-time and historical versions of this information
+-   Allow interested parties to recreate this project
+
+### Features
+
+-   Monitors the ambient temperature, barometric pressure, relative humidity, UV index, ambient light levels and concentration of particles in the air
+-   Calculates the indoor air quality (IAQ) index
+-   Provides both real-time and historical measurements
+-   Communicates data to a client (e.g., a smart phone)
```

Comment:
Is the comma between e.g. and a smart phone necessary?

Reply:
yes

Reply:
I think it would be simpler if it was "Communicates data to the user's device"? Or we could change the Overview and add "through the user's smartphone" and remove this point altogether?

Reply:
This line has been removed anyway as the real-time and historical data access has been summarised

Reply:
Amir, I have reworded  to "Provides both real-time and historical measurements for the user to view on a smart phone"

Reply:
I was thinking something super user-friendly and informal, as non-technical as it can get. I believe a feature of this sort could be as:

- Monitor your house in real-time using our app! 🌟

Reply:
I agree on this point. Makes it more appealing to a more wide base of users.

Reply:
I agree